Lib/test/test_gdb/test_backtrace.py

```diff
@@ -49,6 +49,7 @@ def test_bt_full(self):
 
     @unittest.skipIf(python_is_optimized(),
                      "Python was compiled with optimizations")
+    @unittest.skipIf(support.Py_GIL_DISABLED, "test requires the GIL")
```

Comment:
Maybe it's time to introduce a helper in the `support` package like `requires_gil_enabled`? Actually I don't know how many cases there are  in tests like this, if it's just one case, there's no need for a helper.

Reply:
Including this one, there are six `@unittest.skipIf(support.Py_GIL_DISABLED, ...)`

A helper seems like a reasonable idea to me.

Reply:
> Including this one, there are six `@unittest.skipIf(support.Py_GIL_DISABLED, ...)`
> 
> A helper seems like a reasonable idea to me.

I think this helper should look like this:
```python
def requires_gil_enabled(test):
    if Py_GIL_DISABLED:
        return unittest.skip('needs the GIL enabled')(test)
    return test
```

Reply:
Thanks! I'll add this in a separate PR.